examples/calculations/submit_singlepoint.py

```diff
@@ -0,0 +1,168 @@
+"""Example code for submitting single point calculation"""
+
+from pathlib import Path
+import sys
+from typing import Union
+
+from ase.build import bulk
+from ase.io import read
+import click
+
+from aiida.common import NotExistent
+from aiida.engine import run_get_node
+from aiida.orm import Code, Str, StructureData, load_node
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.data.model import ModelData
+
+# StructureData = DataFactory("core.structure")
+
+
+def load_model(string: Union[str, None], architecture: str) -> ModelData:
```

Comment:
Can `string` be a `Path`?

Something like `model` or `model_path` may also be clearer?